tox.ini

```diff
@@ -52,13 +22,15 @@ whitelist_externals =
     rm
 deps =
     -r{toxinidir}/requirements/doc.txt
+    twine
```

Comment:
Add this in quality.in

Reply:
Agreed, `twine` should be added to the appropriate requirements file.